words

```diff
@@ -1,22 +1,22 @@
 Objective:
-	I’m trying to break out of that catch-22 of needing experience to gain experience.  With ambitions to eventually end up in data sciences, I understand that the path entails a humble beginning and a ton of studying.  I seek long-term mentorship for my skill development in a setting where I am able to grow and meaningfully contribute back.
+	With ambitions to excel in the field of data sciences, I understand that the path entails a humble beginning and a ton of studying.  I seek long-term mentorship for my skill development in a setting where I am able to grow and meaningfully contribute.
```

Comment:
The first line you had wasn't working for me. 
The word "trying" was too telling. No trying. You are doing. The employer will love clear intention, and confidence. I know you don't see yourself as a developer, but you honestly are one, and I think you need to channel one in your writing here. You don't want them to feel like they're taking a chance on you. I feel like you were sending that message. You don't need to. You slay at what you do. Tell them about that! 

You do have experience! "Transferrable" experience is experience 😃